sql-database/create-and-configure-database/create-and-configure-database.sh

```diff
@@ -1,47 +1,56 @@
 #!/bin/bash
 
+# set execution context (if necessary)
+az account set --subscription <replace with your subscription name or id>
+
+# Set the resource group name and location for your server
+export resourceGroupName=myResourceGroup-$RANDOM
+export location=westus2
+
 # Set an admin login and password for your database
 export adminlogin=ServerAdmin
-export password=ChangeYourAdminPassword1
+export password=P@ssw0rd-$RANDOM
```

Comment:
This password style has a very small keyspace, making it susceptible to automated attacks. Use `openssl rand -base64 16` (at minimum) and avoid static tokens that reduce the keyspace.

Other uses of `$RANDOM` are OK, since they're only used for unique identifiers.

Reply:
where is this doc'd in the contributor guide?

Reply:
export password=openssl rand -base64 16
fails

Reply:
searched contributor guide - no information how to use this in our scripts
https://review.docs.microsoft.com/en-us/search/?search=password&scope=Contributor&branch=master

Reply:
also searched all existing cli scripts in the repo - nobody is using openssl rand - please provide example - wasting so much time on this PR this afternoon.....so very frustrating

Reply:
going back to this: export password=<EnterYourComplexPasswordHere>

Reply:
Use ``password=`openssl rand -base64 16` ``.